src/events/events.html

```diff
@@ -0,0 +1,125 @@
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" aria-current="page" href="./events.html">Events</a>
+                </li>
+            </ul>
+            <hr class="d-block d-lg-none mt-0">
+            <ul class="navbar-nav flex-row mb-2 mb-lg-0 gap-2">
+                <li class="nav-item">
+                    <a class="btn btn-primary" href="../sign-in/sign-in.html">Sign in</a>
+                </li>
+                <li class="nav-item">
+                    <a class="btn btn-primary" href="../sign-up/sign-up.html">Sign up</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+
+<main class="main flex-grow-1 bg-primary py-5">
+    <div class="container">
+        <div class="row justify-content-center bg-white p-5 ">
+            <div class="col-lg-6 col-md-12 col-sm-12 col-11 d-flex flex-column justify-content-evenly">
+                <h3 class="text-center mb-4">Welcome to Culinary Excellence Event</h3>
+                <p>
+                    Are you ready to embark on a culinary journey like no other? Our culinary events are specially
+                    designed to inspire and elevate the skills of culinary masters in the art of recipe and menu
+                    management.
+                </p>
+                <ul>
+                    <li><strong>Recipe Creation Workshops:</strong> Join us for hands-on workshops led by renowned
+                        chefs. Learn to craft and perfect your signature recipes while mastering recipe management
+                        techniques that streamline your culinary process.
+                    </li>
+
+                    <li><strong>Menu Planning Masterclass:</strong> Dive deep into the art of menu planning. Discover
+                        how to curate menus that leave a lasting impression, from selecting ingredients to balancing
+                        flavors and creating memorable dining experiences.
+                    </li>
+
+                    <li><strong>Culinary Innovation Symposium:</strong> Join fellow culinary masters, food enthusiasts,
+                        and industry experts for a symposium focused on culinary innovation. Explore emerging trends,
+                        tools, and technologies for recipe and menu management.
+                    </li>
+                </ul>
+                <p>
+                    <strong>Come to the address:</strong> Сквер імені Олени Теліги
+                    вулиця Політехнічна, Київ, 02000
+                </p>
+            </div>
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1385.3026033356023!2d30.454376647419263!3d50.447503622631174!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4cc27725b49d7%3A0xd00801d72ba54c32!2z0KHQutCy0LXRgCDRltC80LXQvdGWINCe0LvQtdC90Lgg0KLQtdC70ZbQs9C4!5e0!3m2!1suk!2sua!4v1696521912163!5m2!1suk!2sua"
```

Comment:
Please, try to keep template clean. Look for other places in the code to see actual code style in the project (it usually can differ)